Source/Rive/Private/Rive/RiveTextureObject.cpp

```diff
@@ -144,7 +151,11 @@ FVector2f URiveTextureObject::GetLocalCoordinatesFromExtents(const FVector2f& In
 
 void URiveTextureObject::Initialize(const FRiveDescriptor& InRiveDescriptor)
 {
-	Artboard = nullptr;
+	if (Artboard == nullptr)
```

Comment:
Let's move the new artboard, and Reinitialization of the artboard into RiveReady.

Reply:
Done.